PowerEditor/src/functionList.xml

```diff
@@ -88,21 +88,21 @@
 						<nameExpr expr="\w+\Z" />
 					</className>
 					<function
-						mainExpr="^\s*((static|public|protected|private|final)*(\s+(static|public|protected|private|final))+\s+)?(function\s+)+(\w+(\s+\w+)?(\s+|\*\s+|\s+\*|\s+\*\s+))?(\w+\s*::)?(?!(if|while|for|switch))[\w~]+\s*\([^\{]*\{"
```

Comment:
On the other hand, invalid function names (reserved word, starts with a digit) can't be used and would crash, so what's the benefit of adding code to hide them? It could even be counterproductive (hide bug).